soh/soh/Enhancements/randomizer/3drando/location_access/locacc_jabujabus_belly.cpp

```diff
@@ -57,7 +57,11 @@ void RegionTable_Init_JabuJabusBelly() {
                   LOCATION(RC_JABU_JABUS_BELLY_GS_LOBBY_BASEMENT_LOWER, logic->HookshotOrBoomerang()),
                   LOCATION(RC_JABU_JABUS_BELLY_GS_LOBBY_BASEMENT_UPPER, logic->HookshotOrBoomerang()),
                   LOCATION(RC_JABU_JABUS_BELLY_GS_WATER_SWITCH_ROOM,    logic->HookshotOrBoomerang()),
-                  //PUT 2 OCTO ROOM POTS HERE
+                  LOCATION(RC_JABU_JABUS_BELLY_TWO_OCTOROK_POT_1, logic->CanBreakPots()),
+                  LOCATION(RC_JABU_JABUS_BELLY_TWO_OCTOROK_POT_2, logic->CanBreakPots()),
+                  LOCATION(RC_JABU_JABUS_BELLY_TWO_OCTOROK_POT_3, logic->CanBreakPots()),
+                  LOCATION(RC_JABU_JABUS_BELLY_TWO_OCTOROK_POT_4, logic->CanBreakPots()),
+                  LOCATION(RC_JABU_JABUS_BELLY_TWO_OCTOROK_POT_5, logic->CanBreakPots()),
```

Comment:
These pots need to have `CanUse(RG_BOOMERANG)` checks.

Reply:
`logic->CanUse(RG_BOOMERANG) || logic->CanUse(RG_HOVER_BOOTS)` actually, mirroring the fairy pot event access.

Reply:
I couldn't get hover boots to work for that access. I did try.

Reply:
> I couldn't get hover boots to work for that access. I did try.


https://github.com/user-attachments/assets/2e42c3d5-b6fe-48f5-af5f-cda05d00d7c9

Unless I'm misunderstanding the room and topic, if so ignore this message, lol